examples/instances_reserved_public_ip/terraform.tfvars.example

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019, 2020 Oracle Corporation and/or affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+/*----------------------------------------------------------------------------
+HOW TO USE THIS FILE
+
+1. keep this file in the same folder as your terraform *.tf files
+2. If your terraform config is managed with Git, add the tfvars file to your .gitignore.
+3. Keep your RSA private key outside of your terraform work folder!
+----------------------------------------------------------------------------*/
+
+# provider identity parameters
+
+tenancy_ocid     = "<tenancy OCID>"
+user_ocid        = "<user OCID>"
+fingerprint      = "<PEM key fingerprint>"
+region           = "<region in which to operate, example: us-ashburn-1, eu-frankfurt-1>"
+private_key_path = "<path to the private key that matches the fingerprint above>"
+
+# general oci parameters
+
+compartment_ocid = "<compartment OCID>"
+
+# compute instance parameters
+
+source_ocid  = "<The OCID of an image or a boot volume>"
+
+# operating system parameters
+
+ssh_authorized_keys = "<path to the instance's public key>"
```

Comment:
This should be a list of keys or list of path to keys. If you intend to add only 1 key for now, then either accept a list and add the 1st key in your code or change the var name to ssh_authorized_key. 

1st one means your interface doesn't change and you can just do a minor release when you want to add support for multiple keys.

2nd one means you'll have to deprecate the singular named variable and then do at least a minor change in the future.

Reply:
see comment [below](https://github.com/oracle-terraform-modules/terraform-oci-compute-instance/pull/71#discussion_r705845730) about plans for ssh_public_key. Let take this change request out of this PR: I will fix it as a dedicated issue, before 2.2.0 release.